clinica/iotools/bids_utils.py

```diff
@@ -818,3 +818,8 @@ def run_dcm2niix(
             ),
             lvl="warning",
         )
+
+
+def write_to_tsv(dataframe: DataFrame, buffer: Union[PathLike, BinaryIO]) -> None:
```

Comment:
I'm getting a NameError on the types here. Are they properly defined?

Reply:
They were not.